flight/test_files/waypoint_pathfinding_test.py

```diff
@@ -187,6 +195,53 @@ def to_svg_coord(x: float, y: float) -> tuple[float, float]:
     return "".join(string_builder)
 
 
+def generate_random_waypoints(
+    vertices: Iterable[Point],
+    utm_zone_number: int,
+    utm_zone_letter: str,
+) -> str:
+    """
+    Generate random waypoints inside a shape that can be used for the
+    waypoint state unit test.
+
+    Parameters
+    ----------
+    vertices : list[Point]
+        The vertices of the boundary of the shape in the correct order.
+    utm_zone_number : int
+        The UTM zone number of the boundary vertices.
+    utm_zone_letter : str
+        The UTM zone letter of the boundary vertices.
+
+    Returns
+    -------
+    str
+        JSON data to be pasted into ../data/waypoint_data.json before
+        running the waypoint state unit test.
+    """
+    waypoints: list[CoordinateWithAltitude] = []
+    for _ in range(100):
+        random_point: Point = random_point_in_shape(vertices)
+
+        latitude: float
+        longitude: float
+        latitude, longitude = utm.to_latlon(
+            random_point.x, random_point.y, utm_zone_number, utm_zone_letter
```

Comment:
Do you have a way to generate points for the golf course?

Reply:
No

Reply:
In line 257 of flight/test_files/waypoint_pathfinding_test, `waypoint_data.json` can be replaced with `golf_data.json`. Running `python3 -m flight.test_files.waypoint_pathfinding_test` will then print 100 random waypoints within the golf course that can then be pasted into flight/data/golf_data.json.